README.md

```diff
@@ -83,19 +98,24 @@ var obj = {
 }
 ```
 
--- YOUR ANSWER HERE --
+- I don't think this is valid JS because the contents of the object are not setup as key:value pairs
+
```

Comment:
This is the new ES6 object shorthand syntax [Method Properties](http://es6-features.org/#MethodProperties)